e2e/app.e2e-spec.ts

```diff
@@ -47,6 +47,43 @@ describe('ng2-date-picker App', () => {
     });
   });
 
+  describe('dpDayPicker reactive directive', () => {
```

Comment:
Those validations won't go through the directive at all - they'll be directly on the `ngModel`.  But I'll put in some tests to make sure that the directive doesn't interfere with those somehow.